configure.ac

```diff
@@ -507,6 +510,9 @@ AC_RUN_IFELSE([
     AC_MSG_RESULT([no])
     AC_MSG_NOTICE([std::regex support appears to be incomplete, fallback to GNU C regex])
     AC_DEFINE([USE_STDLIB_REGEX], [1], [Whether to use GNU C regex])
+],[
+    AC_MSG_NOTICE([Assume boost::regex due to cross-compilation])
```

Comment:
Dependency on boost is completely removed starting 4.26. For regex it is either std::regex or GNU C regex. Please fix a comment.